src/ml_gym/frontend/dashboard/src/features/globalConfig/globalConfig.tsx

```diff
@@ -0,0 +1,39 @@
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import { FilterConfigType } from "../../app/datatypes"
+
+
+type GlobalConfigPropsType = {
+    filterConfig: FilterConfigType
+    setFilterConfig: (filterConfig: FilterConfigType) => void
+}
+
+const GlobalConfig: React.FC<GlobalConfigPropsType> = ({ filterConfig, setFilterConfig }) => {
+
+
+    const handleChange = (event: { target: { value: any; }; }) => {
```

Comment:
you can also use concrete type here. in this case for `handleChange` it would be `React.ChangeEventHandler<FormControlElement>` and for `handleSubmit` -> `React.FormEventHandler<HTMLFormElement>`.

although these are not always the same for different elements.

Reply:
Good point. Thanks :)